custom_theme/boxes.html

```diff
@@ -62,7 +62,10 @@ <h5 class="card-title">{{ box.displayName }}</h5>
                         <span class="badge bg-secondary">{{ tag }}</span>
                       {% endfor %}
                     </p>
-                </div>
+                    <p class="card-text">
```

Comment:
Is an opening `<div>` missing from above this line?

Reply:
Dont think so, this element I added is part of the `card-body`

Reply:
Gotcha...if it's rendering ok then all good 😃